src/pal/src/file/file.cpp

```diff
@@ -1136,14 +1168,17 @@ DeleteFileA(
     ENTRY("DeleteFileA(lpFileName=%p (%s))\n", lpFileName?lpFileName:"NULL", lpFileName?lpFileName:"NULL");
 
     pThread = InternalGetCurrentThread();
-    if (strlen(lpFileName) >= MAX_LONGPATH)
+    length = strlen(lpFileName);
+    if (length >= MAX_LONGPATH)
```

Comment:
It seems that since the lpUnixFileName buffer length is now dynamic, this check is obsolete.